tapeagents/finetune/checkpoints.py

```diff
@@ -356,6 +380,19 @@ def save_tokenizer_only(
         logger.info(f"Save tokenizer to {output_dir}")
         tokenizer.save_pretrained(output_dir)
 
+        # Update config.json if it exists to match tokenizer vocab size
+        config_path = output_dir / "config.json"
```

Comment:
This is incredibly hacky. `save_tokenizer_only` should not be modifying the model configuration. Let us find another solution please.